packages/wrangler/src/queues/cli/commands/consumer/add.ts

```diff
@@ -63,9 +68,33 @@ export async function handler(
 			max_concurrency: args.maxConcurrency,
 		},
 		dead_letter_queue: args.deadLetterQueue,
-	};
+	} as PostConsumerBody;
```

Comment:
Avoid using `as`....

If you want to give the `body` a type, then do: `const body : PostConsumerBody = {...}`